omod-1.12/src/test/java/org/openmrs/module/webservices/rest/web/v1_0/controller/openmrs1_12/OrderGroupController1_12Test.java

```diff
@@ -0,0 +1,159 @@
+		OrderSet orderSet = orderGroup.getOrderSet();
+		Order order = Context.getOrderService().getOrder(1);
+		Order order2 = Context.getOrderService().getOrder(3);
+		List<Order> orders = new ArrayList<Order>();
+		orders.add(order);
+		orders.add(order2);
+		
+		orderGroup.setOrders(orders);
+		order.setOrderGroup(orderGroup);
+		order2.setOrderGroup(orderGroup);
+		
+		MockHttpServletRequest req = request(RequestMethod.GET, getURI() + "/" + RestTestConstants1_12.ORDER_GROUP_UUID);
+		SimpleObject response = deserialize(handle(req));
+		Assert.assertEquals(patient.getUuid(), Util.getByPath(response, "patient/uuid"));
+		Assert.assertEquals(encounter.getUuid(), Util.getByPath(response, "encounter/uuid"));
+		Assert.assertEquals(orderSet.getUuid(), Util.getByPath(response, "orderSet/uuid"));
+		Assert.assertEquals(RestTestConstants1_12.ORDER_GROUP_DISPLAY, Util.getByPath(response, "display"));
+		List<Order> orderList = (List<Order>) PropertyUtils.getProperty(response, "orders");
+		Assert.assertEquals(2, orderList.size());
+	}
+	
+	@Test
+	public void shouldAddOrdersToAnExistingOrderGroup() throws Exception {
+		final String JSON_ORDER = " { \"orders\": [\n" + "    {\n"
+		        + "      \"encounter\": \"" + RestTestConstants1_12.ORDER_GROUP_ENCOUNTER_UUID + "\",\n"
+		        + "      \"orderType\": \"" + RestTestConstants1_12.ORDER_GROUP_ORDER_TYPE_UUID  + "\",\n" + "      \"type\": \"order\",\n"
+		        + "      \"action\": \"NEW\",\n" + "      \"accessionNumber\": \"string\",\n"
+		        + "      \"dateActivated\": \"2012-09-11\",\n" + "      \"scheduledDate\": \"2012-09-11\",\n"
+		        + "      \"patient\": \"" + RestTestConstants1_12.ORDER_GROUP_PATIENT_UUID  + "\",\n"
+		        + "      \"concept\": \"" + RestTestConstants1_12.ORDER_GROUP_CONCEPT_UUID + "\",\n"
+		        + "      \"careSetting\": \"" +  RestTestConstants1_12.ORDER_GROUP_CARESETTING_UUID + "\",\n"
+		        + "      \"autoExpireDate\": \"2012-09-21\",\n"
+		        + "      \"orderer\": \"" + RestTestConstants1_12.ORDER_GROUP_ODERER_UUID + "\",\n" + "      \"previousOrder\": \"\",\n"
+		        + "      \"urgency\": \"ROUTINE\",\n" + "      \"orderReason\": \"\",\n"
+		        + "      \"orderReasonNonCoded\": \"for Test\",\n" + "      \"instructions\": \"string\",\n"
+		        + "      \"commentToFulfiller\": \"string\"\n" + "    }\n" + "  ]\n" + "}";
+		Integer ordersBefore = orderService.getOrderGroupByUuid(RestTestConstants1_12.ORDER_GROUP_UUID).getOrders().size();
+		Assert.assertEquals(0, orderService.getOrderGroupByUuid(RestTestConstants1_12.ORDER_GROUP_UUID).getOrders().size());
+		
+		MockHttpServletRequest req = newPostRequest(getURI() + "/" + RestTestConstants1_12.ORDER_GROUP_UUID, JSON_ORDER);
+		handle(req);
+		Integer ordersAfter = orderService.getOrderGroupByUuid(RestTestConstants1_12.ORDER_GROUP_UUID).getOrders().size();
+		Assert.assertEquals(1, orderService.getOrderGroupByUuid(RestTestConstants1_12.ORDER_GROUP_UUID).getOrders().size());
```

Comment:
Are these two assertions doing anything different?

Reply:
cleared the duplicate